kitsune/wiki/tests/test_templates.py

```diff
@@ -438,6 +438,16 @@ def test_document_view(self):
         eq_(200, response.status_code)
         assert template_used(response, 'wiki/mobile/document.html')
 
+    def test_document_share_link_escape(self):
+        """Ensure that the share link isn't escaped."""
+        r = ApprovedRevisionFactory(
+            content='Test',
+            document__share_link='https://www.example.org',
+        )
+        response = self.client.get(r.document.get_absolute_url())
```

Comment:
Yes, it will, since this is a MobileTestCase, which makes sure that the mobile templates are used.